superbench/benchmarks/micro_benchmarks/gpu_sm_copy_bw_performance.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the GPU SM Copy Bandwidth Performance benchmark."""
+
+import os
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, ReturnCode
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class GpuSmCopyBwBenchmark(MicroBenchmarkWithInvoke):
+    """The GPU SM copy bandwidth performance benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'gpu_sm_copy'
+
+        self.__result_tags = []
+        self.__num_numa_nodes_in_system = len(
+            [x for x in next(os.walk('/sys/devices/system/node/'))[1] if x.startswith('node')]
+        )
+        self.__default_numa_nodes = list(range(self.__num_numa_nodes_in_system))
+        self.__num_gpus_in_system = 0
+        try:
+            self.__num_gpus_in_system = len([x for x in next(os.walk('/sys/class/drm/'))[1] if x.startswith('card')])
+        except StopIteration:
+            logger.info('No GPU found, disable benchmark: {}'.format(self._name))
+        self.__default_gpu_ids = list(range(self.__num_gpus_in_system))
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
+
+        self._parser.add_argument(
+            '--numa_nodes',
+            type=int,
+            nargs='+',
+            default=self.__default_numa_nodes,
+            required=False,
+            help='NUMA nodes to cover.',
+        )
+
+        self._parser.add_argument(
+            '--gpu_ids',
+            type=int,
+            nargs='+',
+            default=self.__default_gpu_ids,
+            required=False,
+            help='Device IDs of GPUs to cover.',
+        )
```

Comment:
I think `gpu_ids` and `numa_nodes` are handled by sb executor. @abuccts

Reply:
Shall we just let users pass `gpu_ids` and `numa_nodes` instead of detecting ourselves?

Reply:
Please refer another benchmark about the numa nodes:
https://github.com/microsoft/superbenchmark/blob/main/superbench/benchmarks/micro_benchmarks/ib_loopback_performance.py#L99

Reply:
Now let executor to force `numactl` and `CUDA_VISIBLE_DEVICES` from outside.